### ISSUE TITLE
Button Force refresh

### ISSUE THREAD

**Issue by petrbures1969** [Hyundai-Kia-Connect/kia_uvo#982]
Would it be possible to add some button to force refresh manually, as when I refresh state in Bluelink? Thanks a lot

**Comment by cdnninja** [comment]
Easy enough for you to add it.

**Comment by cdnninja** [comment]
Yes you can you can also view it in developer tools actions.  I believe referenced in the readme for the title.

**Comment by zulufoxtrot** [comment]
Thanks. Indeed it's `kia_uvo.force_update`

**Comment by Ecirbaf69** [comment]
hello, I'm sorry but I didn't understand where to find this function in H.A
Can you help me ?

**Comment by Ecirbaf69** [comment]
Thanks a lot
